samcli/lib/schemas/schemas_api_caller.py

```diff
@@ -19,58 +19,82 @@ def list_registries(self, next_token=None, limit=10):
         if limit is None:
             limit = 10
         registries = []
-        paginator = self._schemas_client.get_paginator("list_registries")
-        page_iterator = paginator.paginate(
-            PaginationConfig={"StartingToken": next_token, "MaxItems": limit, "PageSize": limit}
-        )
-        page = None
-        for page in page_iterator:
-            for registry in page["Registries"]:
-                registries.append(registry["RegistryName"])
-        if not registries:
-            raise ResourceNotFound("No Registries found. This should not be possible, please raise an issue.")
-        next_token = page.get("NextToken", None)
-        return {"registries": registries, "next_token": next_token}
+        try:
+            paginator = self._schemas_client.get_paginator("list_registries")
+            page_iterator = paginator.paginate(
+                PaginationConfig={"StartingToken": next_token, "MaxItems": limit, "PageSize": limit}
+            )
+            page = None
+            for page in page_iterator:
+                for registry in page["Registries"]:
+                    registries.append(registry["RegistryName"])
+            if not registries:
+                raise ResourceNotFound("No Registries found. This should not be possible, please raise an issue.")
+            next_token = page.get("NextToken", None)
+            return {"registries": registries, "next_token": next_token}
+        except EndpointConnectionError:
+            raise NotAvailableInRegion(
+                "EventBridge Schemas are not available in provided region. "
+                "Please check AWS doc for Schemas supported regions."
```

Comment:
This exact string is hardcoded at lot of places, please extract it to a constant.

Reply:
done